.gitignore

```diff
@@ -1,2 +1,7 @@
+# Generated by MacOS
+.DS_Store
+
+*.xcuserstate
+xcuserdata/
 
```

Comment:
Don't we also want Pods/ here?

Reply:
i think it would be better for a package to contain the Pods as well

Reply:
also the Pods folder was already committed before us, so it would not have any impact to add it in gitignore